edgelet/edgelet-http-workload/src/server/mod.rs

```diff
@@ -49,14 +49,14 @@ impl WorkloadService {
         W: WorkloadConfig + Clone + Send + Sync + 'static,
     {
         let router = router!(
-            get   Version2018_06_28,   "/modules" => Authorization::new(ListModules::new(runtime.clone()), Policy::Anonymous, runtime.clone()),
-            post  Version2018_06_28,   "/modules/(?P<name>[^/]+)/genid/(?P<genid>[^/]+)/sign" => Authorization::new(SignHandler::new(key_store.clone()), Policy::Caller, runtime.clone()),
-            post  Version2018_06_28,   "/modules/(?P<name>[^/]+)/genid/(?P<genid>[^/]+)/decrypt" => Authorization::new(DecryptHandler::new(hsm.clone()), Policy::Caller, runtime.clone()),
-            post  Version2018_06_28,   "/modules/(?P<name>[^/]+)/genid/(?P<genid>[^/]+)/encrypt" => Authorization::new(EncryptHandler::new(hsm.clone()), Policy::Caller, runtime.clone()),
-            post  Version2018_06_28,   "/modules/(?P<name>[^/]+)/certificate/identity" => Authorization::new(IdentityCertHandler::new(hsm.clone(), config.clone()), Policy::Caller, runtime.clone()),
-            post  Version2018_06_28,   "/modules/(?P<name>[^/]+)/genid/(?P<genid>[^/]+)/certificate/server" => Authorization::new(ServerCertHandler::new(hsm.clone(), config), Policy::Caller, runtime.clone()),
+            get   Version2018_06_28,   "/modules" => Authorization::new(ListModules::new(runtime.clone()), Policy::Anonymous),
```

Comment:
Do we need to bump the API version here?

Reply:
Same comment for mgmt apis

Reply:
I think no need for it. Actually, we didn't change anything here for clients of iotedged API, just the way we internally authenticate those clients

Reply:
Yeah, it doesn't need to be changed.